airbyte-integrations/connectors/source-mssql/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.42.4'
+    cdkVersionRequired = '0.43.4'
```

Comment:
I think you meant to change the patch version

Reply:
https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/java/airbyte-cdk/core/src/main/resources/version.properties

I guess the latest has a minor version bump